kem/frodo/frodo640shake/noise.go

```diff
@@ -0,0 +1,18 @@
+package frodo640shake
+
+const CDFTableLen = 13
+
+var CDFTable [CDFTableLen]uint16 = [CDFTableLen]uint16{4643, 13363, 20579, 25843, 29227, 31145, 32103, 32525, 32689, 32745, 32762, 32766, 32767}
+
+func sample(sampled []uint16) {
+	for i := 0; i < len(sampled); i++ {
+		var gaussianSample uint16 = 0
+		sign := sampled[i] & 1
+		unifSample := sampled[i] >> 1
+
+		for j := 0; j < CDFTableLen-1; j++ {
+			gaussianSample += (CDFTable[j] - unifSample) >> 15
+		}
+		sampled[i] = ((-sign) ^ gaussianSample) + sign
```

Comment:
when sign=1, sampled gets `sampled = (0xFF..FF ^ gaussianSample) + 1`
is that final `plus one` part of the result?

Reply:
Yes, since `flipBits(a) + 1 ≡ -a (mod 2^16)` if a is 16 bit number,  this just flips the sign of `gaussianSample` if `sign = 1`. Should I add a comment here?